Program.cs

```diff
@@ -0,0 +1,105 @@
+﻿Console.WriteLine("Введите натуральное число c");
+int c = int.Parse(Console.ReadLine());
+Console.WriteLine("Введите число x");
+double x = double.Parse(Console.ReadLine());
+double rezultat = 1;
+if (c <= 0)
+{
+    Console.WriteLine("Число n неудовлетворяет требованиям, введите другое");
+}
+else
+{
+    for(double k = 1; k<c; k ++)
+    {
+        rezultat = ((k + 1)/(k + 2)) * Math.Pow(x, k);
+    }
+}
+Console.WriteLine(Math.Round(rezultat, 2));
+
+Console.WriteLine("Введите число а, удовлетворяющее условию: 1 < a < 1.5");
+double a = double.Parse(Console.ReadLine());
+double i = 2;
+double rezultatDwa = 1.5;
+if (( a > 1)&(a < 1.5))
+{
+    while(rezultatDwa >= a)
+    {
+        Console.WriteLine(i);
+        i = i +1;
+        rezultatDwa = 1 + 1/i;
+    }
+}

```

Comment:
1. Отделяйте пустой строкой разные по смыслу части кода, например циклы или операторы условия.
2. Учитывайте приоритеты операций и не злоупотребляйте ненужными скобками.
3. Используйте ленивые логические операции.
```suggestion
double rezultatDwa = 1.5;

if (a > 1 && a < 1.5)
{
    while(rezultatDwa >= a)
    {
        Console.WriteLine(i++);
        rezultatDwa = 1 + 1 / i;
    }
}
```